src/components/Onboarding/index.jsx

```diff
@@ -1,5 +1,104 @@
+import styled from "styled-components";
+
+import Button from "../shared/Button";
+
 function Onboarding() {
-  return <h1>Onboarding Page</h1>;
+  return (
+    <Container>
+      <Wrapper className="onboarding-left">
+        <img
+          className="logo-image"
+          src="../../assets/logo_figci.jpg"
+          alt="figci-logo-img"
+          width="150"
+        />
+        <span className="description">
+          피그마 계정으로 로그인하시면 파일버전을 비교해
+          <br />
+          디자인 화면의 변경사항을 쉽게 보여드려요!
+        </span>
+        <Button size="lg">피그마 계정으로 로그인</Button>
+      </Wrapper>
+      <Wrapper className="onboarding-right">
+        <img
+          src="../../assets/onboarding.png"
+          alt="figci-onboarding-img"
+          width="400"
+        />
+        <h1 className="description main">
+          비교할 두 피그마 버전을 알려주시면
+          <br />
+          버전 별 디자인 화면 변경사항을 바로 알려드려요
+        </h1>
+        <span className="description sub">
+          피그마 이전 버전과 최신 버전을 비교해
+          <br />
+          변경사항을 확인해보세요
+        </span>
+      </Wrapper>
+    </Container>
+  );
 }
 
+const Wrapper = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  padding: 80px;
+  border-left: 2px solid #000;
+
+  .description {
+    padding: 24px;
+    margin-bottom: 24px;
+
+    text-align: center;
+    line-height: 30px;
+    font-size: 1.2rem;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 30px;
```

Comment:
`line-height`가 중복으로 들어가 있는 것 같습니다 🥲

```suggestion
    text-align: center;
    font-size: 1.2rem;
    font-style: normal;
    font-weight: 500;
    line-height: 30px;
```

Reply:
수정했습니다!